cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -480,6 +480,52 @@ a|
 Terminate transaction now allows `YIELD`.
 The `WHERE` clause is not allowed on its own, as it is for `SHOW`, but needs the `YIELD` clause.
 
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS [transaction-id[,...]]
+----
+[source, cypher, role="noheader"]
+----
+TERMINATE TRANSACTIONS transaction-id[,...]
+----
+a|
+The `transaction-id` now allows general expressions resolving to a string or a list of strings instead of just parameters.
```

Comment:
```suggestion
The `transaction-id` clause now allows general expressions resolving to a string or a list of strings instead of just parameters.
```

Reply:
My only issue is that adding 'the' would require some reference to what transaction id is. But another way to do this is by simply removing "the" : 
```suggestion
`transaction-id` now allows general expressions resolving to a string or a list of strings instead of just parameters.
```

Reply:
it is The id :P But I can see what you mean